schulcloud/Controllers/Dashboard/CalendarOverviewViewController.swift

```diff
@@ -129,6 +129,10 @@ class CalendarOverviewViewController: UIViewController {
     }
 }
 
-extension CalendarOverviewViewController : ViewControllerHeightDataSource {
+extension CalendarOverviewViewController : ViewHeightDataSource {
     var height: CGFloat { return 200 }
 }
+
+extension CalendarOverviewViewController : PermissionInfoDataSource {
+    static var managedPermissions : UserPermissions { return UserPermissions.calendarView }
```

Comment:
This has to be `requiredPermissions`!

You don't have to use `var` to conform to the protocol. A shorter way to write this would be:
```swift
static let requiredPermissions = UserPermissions.calendarView
```

Reply:
oh ok, that make sense. I got trolled by the refactoring tool. for some reason.